### PR TITLE
Added select with jumps category

### PR DESCRIPTION
**Pull request by Konfeta3-33**
To-do:
- [ ] Skip jumps
- [ ] Display repetitions
- [ ] Cleanup
- [ ] Test it top to bottom

### CODE REVIEW

**Review by skyksandr — APPROVED** [review]

File: app/controllers/students/jumps_controller.rb [hunk 1]
```diff
@@ -46,7 +47,7 @@ def next_category
       return ::Jumps.const_get(params[:type]) if params[:type]
 
       return ::Jumps::Tunnel unless previous_jump
-      return previous_jump.class if previous_jump.repeat?
+      return previous_jump.class if previous_jump.repeat? || !params[:type]
```
Comment (skyksandr) [inline app/controllers/students/jumps_controller.rb:50]: why do we need this `|| !params[:type]`?
This case should be handled 3 lines above by:
```
return ::Jumps.const_get(params[:type]) if params[:type]
```
If `params[:type]` provided this line should not be executed

File: app/controllers/students/jumps_controller.rb [hunk 2]
```diff
@@ -30,7 +30,8 @@ def index
     def update
       authorize @jump
       if @jump.update jump_params
-        redirect_to student_jump_path(@student, next_level(@jump))
+        flash[:notice] = I18n.t('general.changes_updated')
+        redirect_to student_jump_path(@student, type: next_category.to_s)
```
Comment (skyksandr) [inline app/controllers/students/jumps_controller.rb:34]: This gonna fail. you probably meant something different from `student_jump_path` as it won't recognize `type`

**Review by skyksandr — APPROVED** [review]

File: app/controllers/students/jumps_controller.rb (hunk 1 above)
Comment (Konfeta3-33) [reply app/controllers/students/jumps_controller.rb:50]: What about if we don't have `params[:type]`?  For example after Jump E2?

Comment (skyksandr) [reply app/controllers/students/jumps_controller.rb:50]: In that case we need different condition asserting absence of next category

Comment (Konfeta3-33) [reply app/controllers/students/jumps_controller.rb:50]: something like this `return previous_jump.class unless params[:type]`?

**Comment by skyksandr** [comment]
@Konfeta3-33 rubocop is not happy yet

**Pull request merged**